problems/28.Palindrome Number.gs

```diff
@@ -0,0 +1,31 @@
+function isPalindrome(x: number): boolean {
+    if (x < 0) {
+        return false
+    }
+
+    let niz = uNiz(x)
+    let i = 0
+    let j = niz.length - 1
+
+    while (i != j && j > i) {
```

Comment:
Try writing this with For, just to see if you can do it same logic with two counters.

Reply:
ne biva